fcrepo-serialization/src/main/java/org/fcrepo/camel/serialization/SerializationRouter.java

```diff
@@ -91,30 +94,27 @@ public void configure() throws Exception {
             .filter(not(or(header(FCREPO_URI).startsWith(simple("{{audit.container}}/")),
                     header(FCREPO_URI).isEqualTo(simple("{{audit.container}}")))))
             .process(exchange -> {
-                final String baseUrl = exchange.getIn().getHeader(FCREPO_BASE_URL, "", String.class);
                 final String uri = exchange.getIn().getHeader(FCREPO_URI, "", String.class);
-                exchange.getIn().setHeader(FCREPO_IDENTIFIER, uri.replaceAll(baseUrl, ""));
+                exchange.getIn().setHeader(FCREPO_IDENTIFIER, create(uri).getPath());
             })
             .multicast().to("direct:metadata", "direct:binary");
 
         from("direct:metadata")
             .routeId("FcrepoSerializationMetadataUpdater")
-            .to("fcrepo:{{fcrepo.baseUrl}}?accept={{serialization.mimeType}}")
-            .log(INFO, LOGGER,
-                    "Serializing object ${headers[CamelFcrepoIdentifier]}")
-            .setHeader(FILE_NAME)
-                .simple("${headers[CamelFcrepoIdentifier]}.{{serialization.extension}}")
+            .to("fcrepo:localhost?accept={{serialization.mimeType}}")
+            .log(INFO, LOGGER, "Serializing object ${headers[CamelFcrepoIdentifier]}")
+            .setHeader(FILE_NAME).simple("${headers[CamelFcrepoIdentifier]}.{{serialization.extension}}")
             .log(DEBUG, LOGGER, "filename is ${headers[CamelFileName]}")
             .to("file://{{serialization.descriptions}}");
 
         from("direct:binary")
             .routeId("FcrepoSerializationBinaryUpdater")
             .filter().simple("{{serialization.includeBinaries}} == 'true'")
-            .to("fcrepo:{{fcrepo.baseUrl}}?preferInclude=PreferMinimalContainer" +
+            .to("fcrepo:localhost?preferInclude=PreferMinimalContainer" +
                     "&accept=application/rdf+xml")
             .filter().xpath(isBinaryResourceXPath, ns)
             .log(INFO, LOGGER, "Writing binary ${headers[CamelFcrepoIdentifier]}")
-            .to("fcrepo:{{fcrepo.baseUrl}}?metadata=false")
+            .to("fcrepo:localhost?metadata=false")
```

Comment:
What is the rationale for changing this (and two other instances in this class) from `{{fcrepo.baseUrl}}` to `localhost`?